aten/src/ATen/native/BinaryOps.h

```diff
@@ -36,6 +36,13 @@ inline void sub_check(const Tensor& self, const Scalar& scalar) {
               "If you are trying to invert a mask, use the `~` or `logical_not()` operator instead.");
 }
 
+inline void heaviside_check(const Tensor& self, const Tensor& other) {
```

Comment:
this is used in only one place, might as well inline it

Reply:
don't forget to delete it after inlining!

Reply:
Oops! Apologize for my carelessness